{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the <code>django.utils.numberformat.format()</code> function.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-6975\">https://nvd.nist.gov/vuln/detail/CVE-2019-6975</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/dev/releases/security/\">https://docs.djangoproject.com/en/dev/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ\">https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/</a></li>\n<li><a href=\"https://seclists.org/bugtraq/2019/Jul/10\">https://seclists.org/bugtraq/2019/Jul/10</a></li>\n<li><a href=\"https://usn.ubuntu.com/3890-1/\">https://usn.ubuntu.com/3890-1/</a></li>\n<li><a href=\"https://www.debian.org/security/2019/dsa-4476\">https://www.debian.org/security/2019/dsa-4476</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2019/feb/11/security-releases/\">https://www.djangoproject.com/weblog/2019/feb/11/security-releases/</a></li>\n<li><a href=\"https://www.openwall.com/lists/oss-security/2019/02/11/1\">https://www.openwall.com/lists/oss-security/2019/02/11/1</a></li>\n<li><a href=\"https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227\">https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227</a></li>\n<li><a href=\"https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676\">https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676</a></li>\n<li><a href=\"https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3\">https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3</a></li>\n<li><a href=\"https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964\">https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wh4h-v3f2-r2pp\">https://github.com/advisories/GHSA-wh4h-v3f2-r2pp</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp",
            "title": "[django] Uncontrolled Memory Consumption in Django",
            "date_modified": "2023-09-05T18:00:57.000Z",
            "date_published": "2019-02-12T15:36:37.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the <code>django.utils.numberformat.format()</code> function.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-6975\">https://nvd.nist.gov/vuln/detail/CVE-2019-6975</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/dev/releases/security/\">https://docs.djangoproject.com/en/dev/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ\">https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/</a></li>\n<li><a href=\"https://seclists.org/bugtraq/2019/Jul/10\">https://seclists.org/bugtraq/2019/Jul/10</a></li>\n<li><a href=\"https://usn.ubuntu.com/3890-1/\">https://usn.ubuntu.com/3890-1/</a></li>\n<li><a href=\"https://www.debian.org/security/2019/dsa-4476\">https://www.debian.org/security/2019/dsa-4476</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2019/feb/11/security-releases/\">https://www.djangoproject.com/weblog/2019/feb/11/security-releases/</a></li>\n<li><a href=\"https://www.openwall.com/lists/oss-security/2019/02/11/1\">https://www.openwall.com/lists/oss-security/2019/02/11/1</a></li>\n<li><a href=\"https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227\">https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227</a></li>\n<li><a href=\"https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676\">https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676</a></li>\n<li><a href=\"https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3\">https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3</a></li>\n<li><a href=\"https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964\">https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wh4h-v3f2-r2pp\">https://github.com/advisories/GHSA-wh4h-v3f2-r2pp</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp",
            "title": "[django] Uncontrolled Memory Consumption in Django",
            "date_modified": "2023-09-05T18:00:57.000Z",
            "date_published": "2019-02-12T15:36:37.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django 1.11.x before 1.11.19, 2.0.x before 2.0.11, and 2.1.x before 2.1.6 allows Uncontrolled Memory Consumption via a malicious attacker-supplied value to the <code>django.utils.numberformat.format()</code> function.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-6975\">https://nvd.nist.gov/vuln/detail/CVE-2019-6975</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/dev/releases/security/\">https://docs.djangoproject.com/en/dev/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ\">https://groups.google.com/forum/#!topic/django-announce/WTwEAprR0IQ</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/66WMXHGBXD7GSM3PEXVCMCAGLMQYHZCU/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HVXDOVCXLD74SHR2BENGCE2OOYYYWJHZ/</a></li>\n<li><a href=\"https://seclists.org/bugtraq/2019/Jul/10\">https://seclists.org/bugtraq/2019/Jul/10</a></li>\n<li><a href=\"https://usn.ubuntu.com/3890-1/\">https://usn.ubuntu.com/3890-1/</a></li>\n<li><a href=\"https://www.debian.org/security/2019/dsa-4476\">https://www.debian.org/security/2019/dsa-4476</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2019/feb/11/security-releases/\">https://www.djangoproject.com/weblog/2019/feb/11/security-releases/</a></li>\n<li><a href=\"https://www.openwall.com/lists/oss-security/2019/02/11/1\">https://www.openwall.com/lists/oss-security/2019/02/11/1</a></li>\n<li><a href=\"https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227\">https://github.com/django/django/commit/0bbb560183fabf0533289700845dafa94951f227</a></li>\n<li><a href=\"https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676\">https://github.com/django/django/commit/1f42f82566c9d2d73aff1c42790d6b1b243f7676</a></li>\n<li><a href=\"https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3\">https://github.com/django/django/commit/40cd19055773705301c3428ed5e08a036d2091f3</a></li>\n<li><a href=\"https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964\">https://web.archive.org/web/20200227084713/http://www.securityfocus.com/bid/106964</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wh4h-v3f2-r2pp\">https://github.com/advisories/GHSA-wh4h-v3f2-r2pp</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp",
            "title": "[django] Uncontrolled Memory Consumption in Django",
            "date_modified": "2023-09-05T18:00:57.000Z",
            "date_published": "2019-02-12T15:36:37.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wh4h-v3f2-r2pp"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>There is a vulnerability in <code>load()</code> method in definitions/parser.py in the Danijar Hafner definitions package for Python. It can execute arbitrary python commands resulting in command execution.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-20325\">https://nvd.nist.gov/vuln/detail/CVE-2018-20325</a></li>\n<li><a href=\"https://github.com/danijar/definitions/issues/14\">https://github.com/danijar/definitions/issues/14</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-v4x4-98cg-wr4g\">https://github.com/advisories/GHSA-v4x4-98cg-wr4g</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-v4x4-98cg-wr4g",
            "title": "[definitions] Code injection in Danijar Definitions",
            "date_modified": "2023-09-05T17:59:58.000Z",
            "date_published": "2018-12-26T17:45:19.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-v4x4-98cg-wr4g"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 1.10.x before 1.10.8 and 1.11.x before 1.11.5, HTML autoescaping was disabled in a portion of the template for the technical 500 debug page. Given the right circumstances, this allowed a cross-site scripting attack. This vulnerability shouldn't affect most production sites since you shouldn't run with <code>DEBUG = True</code> (which makes this page accessible) in your production settings.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-12794\">https://nvd.nist.gov/vuln/detail/CVE-2017-12794</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-9r8w-6x8c-6jr9\">https://github.com/advisories/GHSA-9r8w-6x8c-6jr9</a></li>\n<li><a href=\"https://usn.ubuntu.com/3559-1/\">https://usn.ubuntu.com/3559-1/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2017/sep/05/security-releases/\">https://www.djangoproject.com/weblog/2017/sep/05/security-releases/</a></li>\n<li><a href=\"https://web.archive.org/web/20170927072701/http://www.securitytracker.com/id/1039264\">https://web.archive.org/web/20170927072701/http://www.securitytracker.com/id/1039264</a></li>\n<li><a href=\"https://web.archive.org/web/20200227150819/http://www.securityfocus.com/bid/100643\">https://web.archive.org/web/20200227150819/http://www.securityfocus.com/bid/100643</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-9r8w-6x8c-6jr9",
            "title": "[django] Django vulnerable to XSS on 500 pages",
            "date_modified": "2023-09-05T17:57:29.000Z",
            "date_published": "2019-01-04T17:50:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-9r8w-6x8c-6jr9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 1.10.x before 1.10.8 and 1.11.x before 1.11.5, HTML autoescaping was disabled in a portion of the template for the technical 500 debug page. Given the right circumstances, this allowed a cross-site scripting attack. This vulnerability shouldn't affect most production sites since you shouldn't run with <code>DEBUG = True</code> (which makes this page accessible) in your production settings.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-12794\">https://nvd.nist.gov/vuln/detail/CVE-2017-12794</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-9r8w-6x8c-6jr9\">https://github.com/advisories/GHSA-9r8w-6x8c-6jr9</a></li>\n<li><a href=\"https://usn.ubuntu.com/3559-1/\">https://usn.ubuntu.com/3559-1/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2017/sep/05/security-releases/\">https://www.djangoproject.com/weblog/2017/sep/05/security-releases/</a></li>\n<li><a href=\"https://web.archive.org/web/20170927072701/http://www.securitytracker.com/id/1039264\">https://web.archive.org/web/20170927072701/http://www.securitytracker.com/id/1039264</a></li>\n<li><a href=\"https://web.archive.org/web/20200227150819/http://www.securityfocus.com/bid/100643\">https://web.archive.org/web/20200227150819/http://www.securityfocus.com/bid/100643</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-9r8w-6x8c-6jr9",
            "title": "[django] Django vulnerable to XSS on 500 pages",
            "date_modified": "2023-09-05T17:57:29.000Z",
            "date_published": "2019-01-04T17:50:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-9r8w-6x8c-6jr9"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>PySAML2 before 5.0.0 does not check that the signature in a SAML document is enveloped and thus signature wrapping is effective, i.e., it is affected by XML Signature Wrapping (XSW). The signature information and the node/object that is signed can be in different places and thus the signature verification will succeed, but the wrong data will be used. This specifically affects the verification of assertions that have been signed.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-5390\">https://nvd.nist.gov/vuln/detail/CVE-2020-5390</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/commit/5e9d5acbcd8ae45c4e736ac521fd2df5b1c62e25\">https://github.com/IdentityPython/pysaml2/commit/5e9d5acbcd8ae45c4e736ac521fd2df5b1c62e25</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/commit/f27c7e7a7010f83380566a219fd6a290a00f2b6e\">https://github.com/IdentityPython/pysaml2/commit/f27c7e7a7010f83380566a219fd6a290a00f2b6e</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/blob/master/CHANGELOG.md#500-2020-01-13\">https://github.com/IdentityPython/pysaml2/blob/master/CHANGELOG.md#500-2020-01-13</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/releases\">https://github.com/IdentityPython/pysaml2/releases</a></li>\n<li><a href=\"https://github.com/IdentityPython/pysaml2/releases/tag/v5.0.0\">https://github.com/IdentityPython/pysaml2/releases/tag/v5.0.0</a></li>\n<li><a href=\"https://pypi.org/project/pysaml2/5.0.0/\">https://pypi.org/project/pysaml2/5.0.0/</a></li>\n<li><a href=\"https://usn.ubuntu.com/4245-1/\">https://usn.ubuntu.com/4245-1/</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/02/msg00025.html\">https://lists.debian.org/debian-lts-announce/2020/02/msg00025.html</a></li>\n<li><a href=\"https://www.debian.org/security/2020/dsa-4630\">https://www.debian.org/security/2020/dsa-4630</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-qf7v-8hj3-4xw7\">https://github.com/advisories/GHSA-qf7v-8hj3-4xw7</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-qf7v-8hj3-4xw7",
            "title": "[pysaml2] Improper Verification of Cryptographic Signature in PySAML2",
            "date_modified": "2023-09-05T17:54:20.000Z",
            "date_published": "2020-05-06T19:41:29.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-qf7v-8hj3-4xw7"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In conference-scheduler-cli, a pickle.load call on imported data allows remote attackers to execute arbitrary code via a crafted .pickle file, as demonstrated by Python code that contains an os.system call.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-14572\">https://nvd.nist.gov/vuln/detail/CVE-2018-14572</a></li>\n<li><a href=\"https://github.com/PyconUK/ConferenceScheduler-cli/issues/19\">https://github.com/PyconUK/ConferenceScheduler-cli/issues/19</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-cf3c-fffp-34qh\">https://github.com/advisories/GHSA-cf3c-fffp-34qh</a></li>\n<li><a href=\"https://joel-malwarebenchmark.github.io/blog/2020/04/25/cve-2018-14572-conference-scheduler-cli/\">https://joel-malwarebenchmark.github.io/blog/2020/04/25/cve-2018-14572-conference-scheduler-cli/</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-cf3c-fffp-34qh",
            "title": "[conference-scheduler-cli] conference-scheduler-cli Arbitrary Code Execution",
            "date_modified": "2023-09-05T15:09:03.000Z",
            "date_published": "2018-10-29T19:05:38.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-cf3c-fffp-34qh"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>NULL Pointer Dereference allows attackers to cause a denial of service (or application crash). This only applies when lxml is used together with libxml2 2.9.10 through 2.9.14. libxml2 2.9.9 and earlier are not affected. It allows triggering crashes through forged input data, given a vulnerable code sequence in the application. The vulnerability is caused by the iterwalk function (also used by the canonicalize function). Such code shouldn't be in wide-spread use, given that parsing + iterwalk would usually be replaced with the more efficient iterparse function. However, an XML converter that serialises to C14N would also be vulnerable, for example, and there are legitimate use cases for this code sequence. If untrusted input is received (also remotely) and processed via iterwalk function, a crash can be triggered.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-2309\">https://nvd.nist.gov/vuln/detail/CVE-2022-2309</a></li>\n<li><a href=\"https://github.com/lxml/lxml/commit/86368e9cf70a0ad23cccd5ee32de847149af0c6f\">https://github.com/lxml/lxml/commit/86368e9cf70a0ad23cccd5ee32de847149af0c6f</a></li>\n<li><a href=\"https://huntr.dev/bounties/8264e74f-edda-4c40-9956-49de635105ba\">https://huntr.dev/bounties/8264e74f-edda-4c40-9956-49de635105ba</a></li>\n<li><a href=\"https://github.com/lxml/lxml/blob/master/CHANGES.txt\">https://github.com/lxml/lxml/blob/master/CHANGES.txt</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/lxml/PYSEC-2022-230.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/lxml/PYSEC-2022-230.yaml</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202208-06\">https://security.gentoo.org/glsa/202208-06</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wrxv-2j5q-m38w\">https://github.com/advisories/GHSA-wrxv-2j5q-m38w</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20220915-0006/\">https://security.netapp.com/advisory/ntap-20220915-0006/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/URHHSIBTPTALXMECRLAC2EVDNAFSR5NO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/URHHSIBTPTALXMECRLAC2EVDNAFSR5NO/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HGYC6L7ENH5VEGN3YWFBYMGKX6WNS7HZ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/HGYC6L7ENH5VEGN3YWFBYMGKX6WNS7HZ/</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wrxv-2j5q-m38w",
            "title": "[lxml] lxml NULL Pointer Dereference allows attackers to cause a denial of service",
            "date_modified": "2023-09-05T14:53:20.000Z",
            "date_published": "2022-07-06T00:00:30.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wrxv-2j5q-m38w"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>typed_ast 1.3.0 and 1.3.1 has a handle_keywordonly_args out-of-bounds read. An attacker with the ability to cause a Python interpreter to parse Python source (but not necessarily execute it) may be able to crash the interpreter process. This could be a concern, for example, in a web-based service that parses (but does not execute) Python code. (This issue also affected certain Python 3.8.0-alpha prereleases.)</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-19274\">https://nvd.nist.gov/vuln/detail/CVE-2019-19274</a></li>\n<li><a href=\"https://github.com/python/cpython/commit/a4d78362397fc3bced6ea80fbc7b5f4827aec55e\">https://github.com/python/cpython/commit/a4d78362397fc3bced6ea80fbc7b5f4827aec55e</a></li>\n<li><a href=\"https://github.com/python/cpython/commit/dcfcd146f8e6fc5c2fc16a4c192a0c5f5ca8c53c\">https://github.com/python/cpython/commit/dcfcd146f8e6fc5c2fc16a4c192a0c5f5ca8c53c</a></li>\n<li><a href=\"https://github.com/python/typed_ast/commit/156afcb26c198e162504a57caddfe0acd9ed7dce\">https://github.com/python/typed_ast/commit/156afcb26c198e162504a57caddfe0acd9ed7dce</a></li>\n<li><a href=\"https://github.com/python/typed_ast/commit/dc317ac9cff859aa84eeabe03fb5004982545b3b\">https://github.com/python/typed_ast/commit/dc317ac9cff859aa84eeabe03fb5004982545b3b</a></li>\n<li><a href=\"https://bugs.python.org/issue36495\">https://bugs.python.org/issue36495</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LG5H4Q6LFVRX7SFXLBEJMNQFI4T5SCEA/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LG5H4Q6LFVRX7SFXLBEJMNQFI4T5SCEA/</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m3jw-62m7-jjcm\">https://github.com/advisories/GHSA-m3jw-62m7-jjcm</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m3jw-62m7-jjcm",
            "title": "[typed-ast] typed-ast Out-of-bounds Read",
            "date_modified": "2023-09-05T14:46:43.000Z",
            "date_published": "2019-12-02T18:02:02.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m3jw-62m7-jjcm"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>typed_ast 1.3.0 and 1.3.1 has an ast_for_arguments out-of-bounds read. An attacker with the ability to cause a Python interpreter to parse Python source (but not necessarily execute it) may be able to crash the interpreter process. This could be a concern, for example, in a web-based service that parses (but does not execute) Python code. (This issue also affected certain Python 3.8.0-alpha prereleases.)</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-19275\">https://nvd.nist.gov/vuln/detail/CVE-2019-19275</a></li>\n<li><a href=\"https://github.com/python/cpython/commit/a4d78362397fc3bced6ea80fbc7b5f4827aec55e\">https://github.com/python/cpython/commit/a4d78362397fc3bced6ea80fbc7b5f4827aec55e</a></li>\n<li><a href=\"https://github.com/python/cpython/commit/dcfcd146f8e6fc5c2fc16a4c192a0c5f5ca8c53c\">https://github.com/python/cpython/commit/dcfcd146f8e6fc5c2fc16a4c192a0c5f5ca8c53c</a></li>\n<li><a href=\"https://github.com/python/typed_ast/commit/156afcb26c198e162504a57caddfe0acd9ed7dce\">https://github.com/python/typed_ast/commit/156afcb26c198e162504a57caddfe0acd9ed7dce</a></li>\n<li><a href=\"https://github.com/python/typed_ast/commit/dc317ac9cff859aa84eeabe03fb5004982545b3b\">https://github.com/python/typed_ast/commit/dc317ac9cff859aa84eeabe03fb5004982545b3b</a></li>\n<li><a href=\"https://bugs.python.org/issue36495\">https://bugs.python.org/issue36495</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LG5H4Q6LFVRX7SFXLBEJMNQFI4T5SCEA/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/LG5H4Q6LFVRX7SFXLBEJMNQFI4T5SCEA/</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-7xxv-wpxj-mx5v\">https://github.com/advisories/GHSA-7xxv-wpxj-mx5v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-7xxv-wpxj-mx5v",
            "title": "[typed-ast] typed-ast Out-of-bounds Read",
            "date_modified": "2023-09-05T14:45:57.000Z",
            "date_published": "2019-12-02T18:03:09.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-7xxv-wpxj-mx5v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The package bottle before 0.12.19 are vulnerable to Web Cache Poisoning by using a vector called parameter cloaking. When the attacker can separate query parameters using a semicolon (;), they can cause a difference in the interpretation of the request between the proxy (running with default configuration) and the server. This can result in malicious requests being cached as completely safe ones, as the proxy would usually not see the semicolon as a separator, and therefore would not include it in a cache key of an unkeyed parameter.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-28473\">https://nvd.nist.gov/vuln/detail/CVE-2020-28473</a></li>\n<li><a href=\"https://github.com/bottlepy/bottle/commit/57a2f22e0c1d2b328c4f54bf75741d74f47f1a6b\">https://github.com/bottlepy/bottle/commit/57a2f22e0c1d2b328c4f54bf75741d74f47f1a6b</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2021/01/msg00019.html\">https://lists.debian.org/debian-lts-announce/2021/01/msg00019.html</a></li>\n<li><a href=\"https://snyk.io/blog/cache-poisoning-in-popular-open-source-packages/\">https://snyk.io/blog/cache-poisoning-in-popular-open-source-packages/</a></li>\n<li><a href=\"https://snyk.io/vuln/SNYK-PYTHON-BOTTLE-1017108\">https://snyk.io/vuln/SNYK-PYTHON-BOTTLE-1017108</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-qhx9-7hx7-cp4r\">https://github.com/advisories/GHSA-qhx9-7hx7-cp4r</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-qhx9-7hx7-cp4r",
            "title": "[bottle] bottle HTTP Request smuggling",
            "date_modified": "2023-09-05T14:37:12.000Z",
            "date_published": "2021-04-07T21:05:21.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-qhx9-7hx7-cp4r"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A XSS vulnerability was discovered in python-lxml's clean module. The module's parser didn't properly imitate browsers, which caused different behaviors between the sanitizer and the user's page. A remote attacker could exploit this flaw to run arbitrary HTML/JS code.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-27783\">https://nvd.nist.gov/vuln/detail/CVE-2020-27783</a></li>\n<li><a href=\"https://github.com/lxml/lxml/commit/a105ab8dc262ec6735977c25c13f0bdfcdec72a7\">https://github.com/lxml/lxml/commit/a105ab8dc262ec6735977c25c13f0bdfcdec72a7</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1901633\">https://bugzilla.redhat.com/show_bug.cgi?id=1901633</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/12/msg00028.html\">https://lists.debian.org/debian-lts-announce/2020/12/msg00028.html</a></li>\n<li><a href=\"https://pypi.org/project/lxml/\">https://pypi.org/project/lxml/</a></li>\n<li><a href=\"https://snyk.io/vuln/SNYK-PYTHON-LXML-1047473\">https://snyk.io/vuln/SNYK-PYTHON-LXML-1047473</a></li>\n<li><a href=\"https://www.debian.org/security/2020/dsa-4810\">https://www.debian.org/security/2020/dsa-4810</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JKG67GPGTV23KADT4D4GK4RMHSO4CIQL/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/JKG67GPGTV23KADT4D4GK4RMHSO4CIQL/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TMHVKRUT22LVWNL3TB7HPSDHJT74Q3JK/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/TMHVKRUT22LVWNL3TB7HPSDHJT74Q3JK/</a></li>\n<li><a href=\"https://advisory.checkmarx.net/advisory/CX-2020-4286\">https://advisory.checkmarx.net/advisory/CX-2020-4286</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210521-0003/\">https://security.netapp.com/advisory/ntap-20210521-0003/</a></li>\n<li><a href=\"https://www.oracle.com//security-alerts/cpujul2021.html\">https://www.oracle.com//security-alerts/cpujul2021.html</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pgww-xf46-h92r\">https://github.com/advisories/GHSA-pgww-xf46-h92r</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pgww-xf46-h92r",
            "title": "[lxml] lxml vulnerable to Cross-site Scripting",
            "date_modified": "2023-09-05T14:36:44.000Z",
            "date_published": "2021-01-07T21:54:01.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pgww-xf46-h92r"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>\"TAXII libtaxii through 1.1.117, as used in EclecticIQ OpenTAXII through 0.2.0 and other products, allows SSRF via an initial http:// substring to the parse method, even when the no_network setting is used for the XML parser. NOTE: the vendor points out that the parse method \"wraps the lxml library\" and that this may be an issue to \"raise ... to the lxml group.\"\"</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-27197\">https://nvd.nist.gov/vuln/detail/CVE-2020-27197</a></li>\n<li><a href=\"https://github.com/TAXIIProject/libtaxii/issues/246\">https://github.com/TAXIIProject/libtaxii/issues/246</a></li>\n<li><a href=\"https://github.com/eclecticiq/OpenTAXII/issues/176\">https://github.com/eclecticiq/OpenTAXII/issues/176</a></li>\n<li><a href=\"https://github.com/TAXIIProject/libtaxii/pull/247\">https://github.com/TAXIIProject/libtaxii/pull/247</a></li>\n<li><a href=\"http://packetstormsecurity.com/files/159662/Libtaxii-1.1.117-OpenTaxi-0.2.0-Server-Side-Request-Forgery.html\">http://packetstormsecurity.com/files/159662/Libtaxii-1.1.117-OpenTaxi-0.2.0-Server-Side-Request-Forgery.html</a></li>\n<li><a href=\"https://github.com/TAXIIProject/libtaxii/commit/23c6f7b69d99e965c8d53dc4710ae64da3fb4842\">https://github.com/TAXIIProject/libtaxii/commit/23c6f7b69d99e965c8d53dc4710ae64da3fb4842</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-836c-xg97-8p4h\">https://github.com/advisories/GHSA-836c-xg97-8p4h</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-836c-xg97-8p4h",
            "title": "[libtaxii] libtaxii Server-Side Request Forgery vulnerability",
            "date_modified": "2023-09-05T14:36:12.000Z",
            "date_published": "2021-04-30T17:34:48.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-836c-xg97-8p4h"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Django Directory Traversal via archive.extract",
            "date_modified": "2023-09-05T14:35:08.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Django Directory Traversal via archive.extract",
            "date_modified": "2023-09-05T14:35:08.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>In Django 2.2 before 2.2.18, 3.0 before 3.0.12, and 3.1 before 3.1.6, the django.utils.archive.extract method (used by \"startapp --template\" and \"startproject --template\") allows directory traversal via an archive with absolute paths or relative paths with dot segments.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3281\">https://nvd.nist.gov/vuln/detail/CVE-2021-3281</a></li>\n<li><a href=\"https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23\">https://github.com/django/django/commit/05413afa8c18cdb978fcdf470e09f7a12b234a23</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/security/\">https://docs.djangoproject.com/en/3.1/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!forum/django-announce\">https://groups.google.com/forum/#!forum/django-announce</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/YF52FKEH5S2P5CM4X7IXSYG67YY2CDOO/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210226-0004/\">https://security.netapp.com/advisory/ntap-20210226-0004/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2021/feb/01/security-releases/\">https://www.djangoproject.com/weblog/2021/feb/01/security-releases/</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/3.1/releases/3.0.12/\">https://docs.djangoproject.com/en/3.1/releases/3.0.12/</a></li>\n<li><a href=\"https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624\">https://github.com/django/django/commit/02e6592835b4559909aa3aaaf67988fef435f624</a></li>\n<li><a href=\"https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37\">https://github.com/django/django/commit/21e7622dec1f8612c85c2fc37fe8efbfd3311e37</a></li>\n<li><a href=\"https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a\">https://github.com/django/django/commit/52e409ed17287e9aabda847b6afe58be2fa9f86a</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-fvgf-6h6h-3322\">https://github.com/advisories/GHSA-fvgf-6h6h-3322</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322",
            "title": "[django] Django Directory Traversal via archive.extract",
            "date_modified": "2023-09-05T14:35:08.000Z",
            "date_published": "2021-03-18T20:29:49.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-fvgf-6h6h-3322"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Django Channels 3.x before 3.0.3 allows remote attackers to obtain sensitive information from a different request scope. The legacy channels.http.AsgiHandler class, used for handling HTTP type requests in an ASGI environment prior to Django 3.0, did not correctly separate request scopes in Channels 3.0. In many cases this would result in a crash but, with correct timing, responses could be sent to the wrong client, resulting in potential leakage of session identifiers and other sensitive data. Note that this affects only the legacy Channels provided class, and not Django's similar ASGIHandler, available from Django 3.0.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-35681\">https://nvd.nist.gov/vuln/detail/CVE-2020-35681</a></li>\n<li><a href=\"https://github.com/django/channels/commit/e85874d9630474986a6937430eac52db79a2a022\">https://github.com/django/channels/commit/e85874d9630474986a6937430eac52db79a2a022</a></li>\n<li><a href=\"https://channels.readthedocs.io/en/stable/releases/3.0.3.html\">https://channels.readthedocs.io/en/stable/releases/3.0.3.html</a></li>\n<li><a href=\"https://channels.readthedocs.io/en/stable/releases/index.html\">https://channels.readthedocs.io/en/stable/releases/index.html</a></li>\n<li><a href=\"https://github.com/django/channels/releases\">https://github.com/django/channels/releases</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-v542-8q9x-cffc\">https://github.com/advisories/GHSA-v542-8q9x-cffc</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-v542-8q9x-cffc",
            "title": "[channels] Django Channels leakage of session identifiers using legacy AsgiHandler",
            "date_modified": "2023-09-05T14:34:16.000Z",
            "date_published": "2021-03-19T21:29:02.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-v542-8q9x-cffc"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An XSS vulnerability was discovered in the python <code>lxml</code> clean module versions before 4.6.3. When disabling <code>the safe_attrs_only</code> and <code>forms</code> arguments, the <code>Cleaner</code> class does not remove the <code>formaction</code> attribute allowing for JS to bypass the sanitizer. A remote attacker could exploit this flaw to run arbitrary JS code on users who interact with incorrectly sanitized HTML. This issue is patched in <code>lxml</code> 4.6.3.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-28957\">https://nvd.nist.gov/vuln/detail/CVE-2021-28957</a></li>\n<li><a href=\"https://github.com/lxml/lxml/pull/316\">https://github.com/lxml/lxml/pull/316</a></li>\n<li><a href=\"https://github.com/lxml/lxml/pull/316/commits/10ec1b4e9f93713513a3264ed6158af22492f270\">https://github.com/lxml/lxml/pull/316/commits/10ec1b4e9f93713513a3264ed6158af22492f270</a></li>\n<li><a href=\"https://github.com/lxml/lxml/commit/2d01a1ba8984e0483ce6619b972832377f208a0d\">https://github.com/lxml/lxml/commit/2d01a1ba8984e0483ce6619b972832377f208a0d</a></li>\n<li><a href=\"https://bugs.launchpad.net/lxml/+bug/1888153\">https://bugs.launchpad.net/lxml/+bug/1888153</a></li>\n<li><a href=\"https://pypi.org/project/lxml\">https://pypi.org/project/lxml</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2021/03/msg00031.html\">https://lists.debian.org/debian-lts-announce/2021/03/msg00031.html</a></li>\n<li><a href=\"https://github.com/lxml/lxml/commit/a5f9cb52079dc57477c460dbe6ba0f775e14a999\">https://github.com/lxml/lxml/commit/a5f9cb52079dc57477c460dbe6ba0f775e14a999</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4880\">https://www.debian.org/security/2021/dsa-4880</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20210521-0004/\">https://security.netapp.com/advisory/ntap-20210521-0004/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3C2R44VDUY7FJVMAVRZ2WY7XYL4SVN45/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/3C2R44VDUY7FJVMAVRZ2WY7XYL4SVN45/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XXN3QPWCTQVOGW4BMWV3AUUZZ4NRZNSQ/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/XXN3QPWCTQVOGW4BMWV3AUUZZ4NRZNSQ/</a></li>\n<li><a href=\"https://www.oracle.com/security-alerts/cpuoct2021.html\">https://www.oracle.com/security-alerts/cpuoct2021.html</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202208-06\">https://security.gentoo.org/glsa/202208-06</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jq4v-f5q6-mjqq\">https://github.com/advisories/GHSA-jq4v-f5q6-mjqq</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jq4v-f5q6-mjqq",
            "title": "[lxml] lxml vulnerable to Cross-Site Scripting ",
            "date_modified": "2023-09-05T14:33:15.000Z",
            "date_published": "2021-03-22T16:53:53.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jq4v-f5q6-mjqq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in Django 2.2 before 2.2.16, 3.0 before 3.0.10, and 3.1 before 3.1.1 (when Python 3.7+ is used). FILE_UPLOAD_DIRECTORY_PERMISSIONS mode was not applied to intermediate-level directories created in the process of uploading files. It was also not applied to intermediate-level collected static directories when using the collectstatic management command.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-24583\">https://nvd.nist.gov/vuln/detail/CVE-2020-24583</a></li>\n<li><a href=\"https://github.com/django/django/commit/8d7271578d7b153435b40fe40236ebec43cbf1b9\">https://github.com/django/django/commit/8d7271578d7b153435b40fe40236ebec43cbf1b9</a></li>\n<li><a href=\"https://docs.djangoproject.com/en/dev/releases/security/\">https://docs.djangoproject.com/en/dev/releases/security/</a></li>\n<li><a href=\"https://groups.google.com/forum/#!topic/django-announce/Gdqn58RqIDM\">https://groups.google.com/forum/#!topic/django-announce/Gdqn58RqIDM</a></li>\n<li><a href=\"https://groups.google.com/forum/#!topic/django-announce/zFCMdgUnutU\">https://groups.google.com/forum/#!topic/django-announce/zFCMdgUnutU</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/F2ZHO3GZCJMP3DDTXCNVFV6ED3W64NAU/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/F2ZHO3GZCJMP3DDTXCNVFV6ED3W64NAU/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OLGFFLMF3X6USMJD7V5F5P4K2WVUTO3T/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/OLGFFLMF3X6USMJD7V5F5P4K2WVUTO3T/</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZCRPQCBTV3RZHKVZ6K6QOAANPRZQD3GI/\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZCRPQCBTV3RZHKVZ6K6QOAANPRZQD3GI/</a></li>\n<li><a href=\"https://security.netapp.com/advisory/ntap-20200918-0004/\">https://security.netapp.com/advisory/ntap-20200918-0004/</a></li>\n<li><a href=\"https://usn.ubuntu.com/4479-1/\">https://usn.ubuntu.com/4479-1/</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2020/sep/01/security-releases/\">https://www.djangoproject.com/weblog/2020/sep/01/security-releases/</a></li>\n<li><a href=\"https://www.openwall.com/lists/oss-security/2020/09/01/2\">https://www.openwall.com/lists/oss-security/2020/09/01/2</a></li>\n<li><a href=\"https://www.oracle.com/security-alerts/cpujan2021.html\">https://www.oracle.com/security-alerts/cpujan2021.html</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m6gj-h9gm-gw44\">https://github.com/advisories/GHSA-m6gj-h9gm-gw44</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m6gj-h9gm-gw44",
            "title": "[django] Django Incorrect Default Permissions",
            "date_modified": "2023-09-05T14:32:19.000Z",
            "date_published": "2021-03-18T20:30:13.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m6gj-h9gm-gw44"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}